sagemaker-pipelines/step-decorator/computer-vision-examples/computer-vision-pipeline.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -r ./requirements.txt
</pre><p> </p><p>Let's append the <code>-U</code> flag here. Otherwise, the pre-installed dependencies e.g. sagemaker won't get updated.</p><p> </p><p>We forgot to do this in the other lightsaber notebooks. I'll take care of the rest of notebooks to add this flag</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Good point, I'll update it.